cl/custom_filters/templatetags/extras.py

```diff
@@ -148,3 +151,63 @@ def sort_caret(request, value) -> SafeString:
         if current.startswith("-"):
             caret = '&nbsp;<i class="gray fa fa-angle-down"></i>'
     return mark_safe(caret)
+
+
+@register.simple_tag
+def citation(obj, fmt=None) -> SafeString:
+    if isinstance(obj, Docket):
+        docket = obj
+        date_of_interest = (
+            docket.date_terminated
+            if docket.date_terminated
+            else docket.date_last_filing
+        )
+        ecf = ""
+    elif isinstance(obj, DocketEntry):
+        docket = obj.docket
+        date_of_interest = obj.date_filed
+        ecf = obj.entry_number
+    else:
+        return mark_safe("Unknown object type")
```

Comment:
I'd sort of rather if this totally crashed so this doesn't get accidentally used somewhere.